src/index.html

```diff
@@ -12,7 +16,69 @@
       href="styles/index.scss"
```

Comment:
The stylesheet link should point to a CSS file, not an SCSS file. Ensure the href attribute points to the compiled CSS file, such as 'styles/index.css'.